case-lib/lib.sh

```diff
@@ -343,3 +343,26 @@ disable_kernel_check_point()
 {
     KERNEL_CHECKPOINT="disabled"
 }
+
+is_zephyr()
+{
+    # check if jq is installed, will remove this part
+    # after all DUTs have jq installed.
+    hash jq 2>/dev/null || sudo apt install jq -y &>/dev/null
```

Comment:
Nit: with `type -p` instead of `hash` you can redirect only stdout and not redirect stderr which is "safer".

There's no need to redirect the output of `apt install`, it's not very verbose, it's good to see when it happened and it's even more important to see any error like proxy configuration or network glitch or dependency problem - many things could go wrong. This is test code, the output does not have to be very "tidy", seeing what's going on is much more important.